docs/Frameworks.md

```diff
@@ -53,6 +53,100 @@ it('should test something', () => {
 
 If you want to run something asynchronously, you can either use the [`browser.call`](call) command or [custom commands](CustomCommands.md).
 
+### Mocha Options
+
+The following options can be applied in your `wdio.conf.js` to configure your Mocha environment:
+
+### require
+The `require` option is useful when you want to add or extend some basic functionality (WebdriverIO framework option).
+
+Type: `string|string[]`<br>
+Default: `[]`
+
+### compilers
```

Comment:
Do you think it makes sense removing compilers due to deprecation ? 
https://github.com/mochajs/mocha/wiki/compilers-deprecation

Reply:
I was thinking of that too but it would require us to deprecate it first and then remove it with the next major version release. Therefor I propose:

- merge this PR as it is mainly about documenting these framework options
- make a new PR that removes the documentation (all appearances) and implement deprecation warning
- make a new PR and remove everything completely